src/Databases/PostgreSQL/FetchFromPostgreSQL.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#if !defined(ARCADIA_BUILD)
+#include "config_core.h"
+#endif
+
+#if USE_LIBPQXX
+#include <Storages/StoragePostgreSQL.h>
+
+namespace DB
+{
+
+std::shared_ptr<NamesAndTypesList> fetchTableStructure(ConnectionPtr connection, const String & postgres_table_name, bool use_nulls);
```

Comment:
These functions have too generic names but they are applicable only for PostgreSQL.
I'm surprised that there are no another function with `DB::getDataType` name.

Reply:
Renamed.